src/BPEssentials/Commands/Player/SetJob.cs

```diff
@@ -3,20 +3,47 @@
 using BrokeProtocol.Entities;
 using BrokeProtocol.Utility;
 using BrokeProtocol.Required;
+using System.Collections.Generic;
+using System;
+using System.Linq;
+using BPEssentials.Utils;
 
 namespace BPEssentials.Commands
 {
     public class SetJob : Command
     {
-        public void Invoke(ShPlayer player, JobIndex job, ShPlayer target = null)
+        public void Invoke(ShPlayer player, string jobName, ShPlayer target = null)
         {
             
             target = target ?? player;
-            target.svPlayer.SvTrySetJob(job, true, false);
-            player.TS("job_set", target.username.CleanerMessage(), job.ToString());
+            Dictionary<string, JobIndex> jobs = new Dictionary<string, JobIndex>();
```

Comment:
Use `var` wherever possible.
```suggestion
            var jobs = new Dictionary<string, JobIndex>();
```